### ISSUE TITLE
Some global properties do not work.

### ISSUE THREAD

**Issue by spyse** [json-editor/json-editor#836]
### General information

- json-editor version: 2.4.0

### Expected behavior
With 
`JSONEditor.defaults.editors.object.options.collapsed` = true;`
I would expect to have all objects collapsed.

((( object_layout seems also to not work, although I have it as a option on an edit instance, not globally )))

### Actual behavior

But I get the following error in the console:

`Uncaught TypeError: Cannot set property 'collapsed' of undefined`

### Steps to reproduce the behavior

I built a local index.html with the necessary JS-Files and this minimal Code

```javascript
   const container = document.getElementById('jsonEditorContainer');

    const tcmSchema = ...;
    
    const editor = new JSONEditor(container, {
      schema: JSON.parse(tcmSchema),
      theme: 'bootstrap4',
      disable_edit_json: true,
      disable_properties: true,
      array_controls_top: true,
      iconlib: 'fontawesome5'
    });

    JSONEditor.defaults.editors.object.options.collapsed = true;
```

**Comment by germanbisurgi** [comment]
The readme is wrong. JSONEditor.defaults.editors is a list of editor classes  and it is not possible to set global variables there. We should find a way to do this and remove that fake news from the readme

**Comment by germanbisurgi** [comment]
At second glance, the readme entry predates the problem and is correct. The problem started after a major refactoring from ES15 to ES16.

Before the refactoring, an `options` object was set as the default property for each editor. After the refactoring, the list of editors was simply empty and nothing was set.

Should bi fixed in #1200
